### DISCUSSION TITLE
Drawing fonction not enabled

### DISCUSSION THREAD

**Discussion by melaboudi** [mithnusa/odoo-google-maps#26] [Q&A]
J'ai besoin d'utiliser la fonction 
![draw](https://github.com/mithnusa/odoo-addons/assets/3328348/53dce908-6e3d-40d6-94e1-edbec5ca9485)
mais il n'est pas disponible sur mon instalation. Comment activer cette fonctionalité

**Comment by gityopie** [comment]
Hi @melaboudi 
It's a demo module to show case the Google Maps Drawing feature
You can find the module inside this module https://github.com/mithnusa/odoo-addons/tree/16.0/web_view_google_map_drawing
<img width="762" alt="Screenshot 2023-08-23 at 23 10 13" src="https://github.com/mithnusa/odoo-addons/assets/3539144/a1b9db9f-6c04-4dd5-bada-bbc5ebc5f269">

**Comment by melaboudi** [comment]
![image](https://github.com/mithnusa/odoo-addons/assets/3328348/cada55e4-0a21-43b1-845b-fa41a91da2cf)

**Comment by melaboudi** [comment]
but not work

**Comment by melaboudi** [comment]
ok good job
